src/components/PublicBeat.tsx

```diff
@@ -17,36 +24,51 @@ const useStyles = makeStyles(() => ({
     position: 'relative',
     display: 'inline-block',
     margin: 20,
+    cursor: 'pointer'
   },
   background: {
     backgroundRepeat: 'no-repeat',
-    width: 300,
-    height: 250,
+    width: 200,
+    height: 150,
     opacity: 0.4
   },
   bottomLeftCorner: {
     position: 'absolute',
-    bottom: 0
+    bottom: 0,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'flex-end'
   },
   beatPicture: {
     width: 75,
     height: 75,
+  },
+  playButtonAndBeatInfo: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  playButton: {
+    cursor: 'pointer'
   }
 }));
 
-const PublicBeat: React.FC = () => {
+const PublicBeat: React.FC<PublicBeatProps> = ({...props}) => {
   const classes = useStyles();
+  
+  const musicProvider = React.useContext(MusicContext);
 
   let beats = [
     {
+      'id': '6',
       'picture': 'https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/African_Bush_Elephant.jpg/440px-African_Bush_Elephant.jpg',
       'background': 'https://img-aws.ehowcdn.com/560x560p/s3-us-west-1.amazonaws.com/contentlab.studiod/getty/aac4f9b5127946ec8cc85c718d4261d5',
       'title': 'guitar testing title',
-      'type': 'abc',
+      'type': 'abcabcabcabcabcabcabcabcabc',
       'numBeats': 10,
```

Comment:
This seems like test data. Usually better to store this in an external JSON file to make it easier to understand the actual code